File: SecondYear/any-mesh_AnimeshKumarSingh_2226CSE(AI&ML)1119_2/Week_3/selectionSort.cpp
#include <bits/stdc++.h> 
void selectionSort(vector<int>& arr, int n)
{   
    // Write your code here.
    for (int i = 0; i < n - 1; i++) {
        int min = i;
        for (int j = i + 1; j < n; j++) {
            if (arr[j] < arr[min])
                min = j;
        }

        if (min != i)
            swap(arr[min], arr[i]);
    }
}

File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_3/selectionsort.java
public class selectionsort {
	public static void selectionSort(int arr[], int n) {
		// Write your code here.
		for(int i =0; i<arr.length-1;i++){
			int min = arr[i];
			int minIndex = i;
			for(int j = i+1; j<arr.length; j++){
				if(arr[j]<min){
					min = arr[j];
					minIndex = j;
				}
			}
			int temp = arr[i];
			arr[i] = arr[minIndex];
			arr[minIndex] = temp;
		}
	}
}